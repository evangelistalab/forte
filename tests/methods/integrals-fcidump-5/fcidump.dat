molecule {
0 1
H
H 1 R
R = 1.5
units bohr
symmetry c1
}

set {
  basis sto-3g
  scf_type pk
  e_convergence 14
}

e, wfn = energy('fci',return_wfn=True)
fcidump(wfn,oe_ints=['EIGENVALUES'])
