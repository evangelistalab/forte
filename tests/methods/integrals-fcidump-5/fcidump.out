
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 undefined 

                         Git: Rev {fcidump_sym} 00c99be dirty


    R. M. Parrish, L. A. Burns, D. G. A. Smith, A. C. Simmonett,
    A. E. DePrince III, E. G. Hohenstein, U. Bozkaya, A. Yu. Sokolov,
    R. Di Remigio, R. M. Richard, J. F. Gonthier, A. M. James,
    H. R. McAlexander, A. Kumar, M. Saitow, X. Wang, B. P. Pritchard,
    P. Verma, H. F. Schaefer III, K. Patkowski, R. A. King, E. F. Valeev,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, and C. D. Sherrill,
    J. Chem. Theory Comput. 13(7) pp 3185--3197 (2017).
    (doi: 10.1021/acs.jctc.7b00174)


                         Additional Contributions by
    P. Kraus, H. Kruse, M. H. Lechner, M. C. Schieber, R. A. Shaw,
    A. Alenaizan, R. Galvelis, Z. L. Glick, S. Lehtola, and J. P. Misiewicz

    -----------------------------------------------------------------------


    Psi4 started on: Monday, 06 July 2020 08:58PM

    Process ID: 93238
    Host:       CHE-FEVANGE-01I (2)
    PSIDATADIR: /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule {
0 1
H
H 1 R
R = 1.5
units bohr
symmetry c1
}

set {
  basis sto-3g
  scf_type pk
  e_convergence 14
}

e, wfn = energy('fci',return_wfn=True)
fcidump(wfn,oe_ints=['EIGENVALUES'])
--------------------------------------------------------------------------

Scratch directory: /tmp/

*** tstart() called on CHE-FEVANGE-01I (2)
*** at Mon Jul  6 20:58:40 2020

   => Loading Basis Set <=

    Name: STO-3G
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-2 entry H          line    19 file /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4/basis/sto-3g.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: D_inf_h

    Geometry (in Bohr), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         H            0.000000000000     0.000000000000    -0.750000000000     1.007825032230
         H            0.000000000000     0.000000000000     0.750000000000     1.007825032230

  Running in c1 symmetry.

  Rotational constants: A = ************  B =     53.09536  C =     53.09536 [cm^-1]
  Rotational constants: A = ************  B = 1591758.83927  C = 1591758.83927 [MHz]
  Nuclear repulsion =    0.666666666666667

  Charge       = 0
  Multiplicity = 1
  Electrons    = 2
  Nalpha       = 1
  Nbeta        = 1

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-14
  Density threshold  = 1.00e-08
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: STO-3G
    Blend: STO-3G
    Number of shells: 2
    Number of basis function: 2
    Number of Cartesian functions: 2
    Spherical Harmonics?: true
    Max angular momentum: 0

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   2
      Number of AO shells:               2
      Number of primitives:              6
      Number of atomic orbitals:         2
      Number of basis functions:         2

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 12 doubles for integral storage.
  We computed 6 shell quartets total.
  Whereas there are 6 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 3.7515623104E-01.
  Reciprocal condition number of the overlap matrix is 2.3088757098E-01.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     A          2       2 
   -------------------------
    Total       2       2
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter SAD:    -0.70233159926131   -7.02332e-01   0.00000e+00 
   @RHF iter   1:    -1.11169589336866   -4.09364e-01   1.00550e-16 DIIS
   @RHF iter   2:    -1.11169589336866   -4.44089e-16   0.00000e+00 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A     -0.556924  

    Virtual:                                                              

       2A      0.618390  

    Final Occupation by Irrep:
              A 
    DOCC [     1 ]

  @RHF Final Energy:    -1.11169589336866

   => Energetics <=

    Nuclear Repulsion Energy =              0.6666666666666666
    One-Electron Energy =                  -2.4428764829334071
    Two-Electron Energy =                   0.6645139228980764
    Total Energy =                         -1.1116958933686640

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.0000

  Electronic Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:    -0.0000

  Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000

  Dipole Moment: [D]
     X:     0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000


*** tstop() called on CHE-FEVANGE-01I (2) at Mon Jul  6 20:58:40 2020
Module time:
	user time   =       0.27 seconds =       0.00 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.27 seconds =       0.00 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
 MINTS: Wrapper to libmints.
   by Justin Turney

   Calculation information:
      Number of threads:                 1
      Number of atoms:                   2
      Number of AO shells:               2
      Number of SO shells:               2
      Number of primitives:              6
      Number of atomic orbitals:         2
      Number of basis functions:         2

      Number of irreps:                  1
      Integral cutoff                 0.00e+00
      Number of functions per irrep: [   2 ]

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

      Computing two-electron integrals...done
      Computed 6 non-zero two-electron integrals.
        Stored in file 33.


         ---------------------------------------------------------
                          Configuration Interaction
                            (a 'D E T C I' module)

                 C. David Sherrill, Daniel G. A. Smith, and
                              Matt L. Leininger
         ---------------------------------------------------------


   ==> Parameters <==

    EX LEVEL       =        2      H0 BLOCKSIZE  =     1000
    VAL EX LEVEL   =        0      H0 GUESS SIZE =     1000
    H0COUPLINGSIZE =        0      H0 COUPLING   =       NO
    MAXITER        =       24      NUM PRINT     =       20
    NUM ROOTS      =        1      ICORE         =        1
    PRINT LVL      =        1      FCI           =      YES
    R CONV         = 1.00e-04      MIXED         =      YES
    E CONV         = 1.00e-14      MIXED4        =      YES
    R4S            =       NO      REPL OTF      =       NO
    DIAG METHOD    =      SEM      FOLLOW ROOT   =        0
    PRECONDITIONER = DAVIDSON      UPDATE        = DAVIDSON
    S              =   0.0000      Ms0           =      YES
    GUESS VECTOR   =  H0BLOCK      OPENTYPE      =     NONE
    COLLAPSE SIZE  =        1      HD AVG        = EVANGELISTI
    MAX NUM VECS   =       25      REF SYM       =     AUTO
    IOPEN        =       NO

    EX ALLOW       =  1  1 
    STATE AVERAGE  =  0(1.00) 

   ==> CI Orbital and Space information <==

   ------------------------------------
               Space    Total     A
   ------------------------------------
                 Nso        2     2
                 Nmo        2     2
               Ndocc        1     1
               Nsocc        0     0
   ------------------------------------
              CI Spaces
   ------------------------------------
        Dropped DOCC        0     0
              Active        2     2
        Dropped UOCC        0     0
   ------------------------------------

   ==> Setting up CI strings <==

    There are 2 alpha and 2 beta strings
    The CI space requires 4 (4.00E+00) determinants and 1 blocks

   ==> Transforming CI integrals <==

	Presorting SO-basis two-electron integrals.
	Sorting File: SO Ints (nn|nn) nbuckets = 1
	Transforming the one-electron integrals and constructing Fock matrices
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.

   ==> Starting CI iterations <==

    H0 Block Eigenvalue =  -1.11169589

    Simultaneous Expansion Method (Block Davidson Method)
    Using 1 initial trial vectors

     Iter   Root       Total Energy       Delta E      C RMS

   @CI  0:     0     -1.111695893369   -1.7784E+00   1.8427E-01  
   @CI  1:     0     -1.134690658798   -2.2995E-02   2.6356E-16  
    Warning: Norm of correction (root 0) is < 1.0E-13
   @CI  2:     0     -1.134690658798   0.0000E+00   2.6559E-16 c

   ==> Energetics <==

    SCF energy =           -1.111695893368664
    Total CI energy =      -1.134690658798176

   ==> FCI root 0 information <==

    FCI Root 0 energy =    -1.134690658798176

   The 4 most important determinants:

    *   1   -0.992303  (    0,    0)  1AX 
    *   2    0.123830  (    1,    1)  2AX 

		 "A good bug is a dead bug" 

			 - Starship Troopers

		 "I didn't write FORTRAN.  That's the problem."

			 - Edward Valeev
Writing integrals in FCIDUMP format to INTDUMP
 MINTS: Wrapper to libmints.
   by Justin Turney

   Calculation information:
      Number of threads:                 1
      Number of atoms:                   2
      Number of AO shells:               2
      Number of SO shells:               2
      Number of primitives:              6
      Number of atomic orbitals:         2
      Number of basis functions:         2

      Number of irreps:                  1
      Integral cutoff                 0.00e+00
      Number of functions per irrep: [   2 ]

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

      Computing two-electron integrals...done
      Computed 6 non-zero two-electron integrals.
        Stored in file 33.

	Presorting SO-basis two-electron integrals.
	Sorting File: SO Ints (nn|nn) nbuckets = 1
	Transforming the one-electron integrals and constructing Fock matrices
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.
Integral transformation complete!
Writing TEI integrals in FCIDUMP format to INTDUMP
Writing frozen core operator in FCIDUMP format to INTDUMP
Writing orbital energies in FCIDUMP format to INTDUMP
Writing frozen core + nuclear repulsion energy in FCIDUMP format to INTDUMP
Done generating INTDUMP with integrals in FCIDUMP format.

    Psi4 stopped on: Monday, 06 July 2020 08:58PM
    Psi4 wall time for execution: 0:00:00.43

*** Psi4 exiting successfully. Buy a developer a beer!
